plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.xml

```diff
@@ -118,7 +118,7 @@
 		  scope="total"
 		  class="com.google.cloud.tools.eclipse.appengine.validation.AppEngineWebXmlSourceValidator"
 		  id="appEngineWebXmlSourceValidator">
-		<contentTypeIdentifier id="appengine-web.xml">
+		<contentTypeIdentifier id="com.google.appengine.dd.xml">
```

Comment:
Per https://cloud.google.com/appengine/docs/standard/java/config/appref it's called a "configuration file".